tests/test_cli.sh

```diff
@@ -23,8 +23,8 @@ function setup_tests {
     # do a backup of the original blueprints directory and get rid of the git
     # directory (otherwise all of the initial changes in blueprints would have
     # to be done using blueprints push)
-    cp -r $BLUEPRINTS_DIR ${BLUEPRINTS_DIR}.orig
-    rm -rf $BLUEPRINTS_DIR/git
+    cp -r $blueprints_dir ${blueprints_dir}.orig
+    rm -rf $blueprints_dir/git
```

Comment:
Would be nice to squash this into the previous commit to avoid an untidy git history.